apps/web/src/locales/en/messages.json

```
@@ -563,6 +566,12 @@
   "deletedFolder": {
     "message": "Folder deleted"
   },
+  "editInfo": {
+    "message": "Edit Info"
```

question: Aren't we moving away from sentence case?
```suggestion
    "message": "Edit info"
```